support/appveyor.yml

```diff
@@ -1,22 +1,54 @@
-configuration:
-  - Debug
-  - Release
+configuration: Debug
```

Comment:
I think we should build both in Debug and Release modes.

Reply:
That'll increase the build time from ~30 mins to an hour (6 -> 12 jobs, 5 min/job), but sure. I think we should look into some ways to decrease this time.